### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by rhinof**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (vladshub) [inline .github/ISSUE_TEMPLATE/bug_report.md]: You should remove the device behavior

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (vladshub) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Logs instead of screenshots?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (rhinof) [reply .github/ISSUE_TEMPLATE/bug_report.md]: removed the screenshots section and added a logs section

Comment (rhinof) [reply .github/ISSUE_TEMPLATE/bug_report.md]: In addition removed the device section

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (rhinof) [reply .github/ISSUE_TEMPLATE/bug_report.md]: removed screenshots section and added logs section

**Pull request merged**